v23.1/create-schedule-for-changefeed.md

```diff
@@ -0,0 +1,154 @@
+---
+title: CREATE SCHEDULE FOR CHANGEFEED
+summary: The CREATE SCHEDULE FOR CHANGEFEED statement creates a schedule for periodic changefeeds.
+toc: true
+docs_area: reference.sql
+---
+
+{% include_cached new-in.html version="v23.1" %} `CREATE SCHEDULE FOR CHANGEFEED` allows you to create a scheduled changefeed to export data out of CockroachDB. Scheduled changefeeds have the scale, observability, and endpoint sink options that changefeed jobs include, with the convenience of setting a regular schedule. A changefeed job created with `CREATE SCHEDULE FOR CHANGEFEED` performs a one-time table scan using the [initial scan](create-changefeed.html#initial-scan) functionality to create an export of your table data.
+
+For more detail on using changefeeds to create an export of your table data, see [Export Data with Changefeeds](export-data-with-changefeeds.html).
+
+## Required privileges
+
+{% include {{ page.version.version }}/cdc/privilege-model.md %}
```

Comment:
These privilege sections are pulled in from the `CREATE CHANGEFEED` page, which was updated with the permission work in v22.2. There may be some changes needed to this that I will complete with the docs feature work for 23.1 fine-grained permissions in Feb/Mar while the 23.1 docs are still in `dev`.